boards/arm/nucleo_l4r5zi/nucleo_l4r5zi_defconfig

```diff
@@ -24,6 +24,9 @@ CONFIG_PINMUX=y
 # Enable GPIO
 CONFIG_GPIO=y
 
+# Enable LPTIM
+CONFIG_STM32_LPTIM_TIMER=y
```

Comment:
I'd prefer we keep LPTIM opionnal for now.
Could be enabled in a dedicated sample though